IDEAS/Buildings/Components/InternalGains/Simple.mo

```diff
@@ -0,0 +1,88 @@
+within IDEAS.Buildings.Components.InternalGains;
+model Simple
+  "Simple occupancy model with constant levels of sensible, latent and CO2 heat production per person"
```

Comment:
That's not the intended use. A more complicated model that replaces `Simple` may be to take into account that the latent/sensible fraction depends on the air temperature. Right now this fraction is fixed.
I both cases the parameters should be in the occupancy-type so the description you propose would not really allow to make a distinction between the two?